src/image_builder.py

```diff
@@ -301,6 +315,10 @@ def image_builder(buildspec, image_types=[], device_types=[]):
         PRE_PUSH_STAGE_IMAGES.append(pre_push_stage_image_object)
         FORMATTER.separator()
 
+    if utils.is_APatch_build():
+        FORMATTER.banner("APATCH-PREP")
+        initiate_multithreaded_autopatch_prep(PRE_PUSH_STAGE_IMAGES, make_dummy_boto_client=True)
```

Comment:
Add comment: "It should not be done while other multi-threaded step is in progress"